src/views/network/VpcTab.vue

```diff
@@ -130,6 +130,12 @@
                   v-decorator="['vlan', {rules: [{ required: true, message: `${this.$t('label.required')}` }]}]"
                 ></a-input>
               </a-form-item>
+              // TODO : Add version check
+              <a-form-item :label="$t('label.bypassvlanoverlapcheck')">
```

Comment:
Add check to see if API param is available, @davidjumani see https://github.com/apache/cloudstack-primate/commit/e606fe93560bc7b7d45baaa7b9193a4d60ab8b86